utils/shooter/firmware/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+include_directories(${ROS_LIB_DIR})
+
+# Remove this if using an Arduino without native USB (eg, other than Leonardo)
+add_definitions(-DUSB_CON)
+
+#print_board_list()
+#print_programmer_list()
+
+generate_arduino_firmware(arduino
+  SRCS shooter.cpp ${ROS_LIB_DIR}/time.cpp
+  BOARD atmega328
+  PORT /dev/serial/by-id/usb-FTDI_FT232R_USB_UART_A603UWY5-if00-port0
```

Comment:
Goood good always by the id. We will need to make some udev rules (which let us automatically name and give permissions to the usb when it plugged in) for the arduino, I will send you a good tutorial for how to do it, but this will work for now